### PR TITLE
Move old docs to /legacy and add notice

### PR DESCRIPTION
**Pull request by jacobpgn**
@harrison Could you review please?

This moves everything into the /legacy folder, should add redirects for old root routes, and makes the purpose of this documentation a bit clearer:

screen shot 2016-04-04 at 19 01 03

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ var reload = browserSync.reload;
 var runSequence = require('run-sequence');
 var swig = require('swig');
 
-var dest = process.env.OUTPUT || "_site";
+var dest = process.env.OUTPUT || "_site/legacy";
```
Comment (harrison) [inline gulpfile.js]: script/build also needs `/legacy`, maybe we should put in some shared config

**Comment by harrison** [comment]
Nice, approach makes sense.

Deployed to: https://developer-staging.gocardless.com/legacy/, not sure what's causing the missing files

`npm start` should open `/legacy`, think under `browserSync`

**Comment by jacobpgn** [comment]
@harrison Updated `build` and `npm start`. Deploy looks okay to me, perhaps it was just cache related?

Only annoying thing is that I don't think it'll write the root redirect, as there's the old root index there... Might have to manually add that redirect until we replace it with the new docs :disappointed:

**Comment by harrison** [comment]
:+1: and add the root redirect when doing pro

**Comment by jacobpgn** [comment]
Let's do this!

**Pull request merged**